frontend/src/components/dialog/CreatePeriodDialog.vue

```diff
@@ -1,12 +1,14 @@
 <template>
   <dialog-form
-    v-model="visible"
     icon="mdi-calendar-plus"
     title="Create Period"
     max-width="600px"
     :submit-action="createPeriod"
     submit-color="success"
-    :cancel-action="close">
+    :cancel-action="close"
+    :value="value"
```

Comment:
Ist diese Zeile nicht schon automatisch in `v-bind="$attrs"` impliziert?

Reply:
Ist nach dem dritten Commit wieder mit `v-model="showDialog"` umgesetzt und daher eh weg. Ich hatte allerdings manchmal Mühe, dass value mit $attrs nicht mitgekommen ist. Wieso genau bin ich aber nicht auf die Schliche gekommen. Habe dann bei Codes von anderen gesehen, dass sie `:value`  noch explizit listen.